### PR TITLE
X86 SIMD Support

### PR DESCRIPTION
**Pull request by fengye2two**

### CODE REVIEW

**Comment by tencent-adm** [comment]
[CLA assistant check](https://cla.opensource.tencent.com/Tencent/ncnn?pullRequest=1248) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla.opensource.tencent.com/Tencent/ncnn?pullRequest=1248) before we can accept your contribution.<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla.opensource.tencent.com/check/Tencent/ncnn?pullRequest=1248) it.</sub>

**Comment by nihui** [comment]
please fix ci compilation error

File: .gitignore [hunk 1]
```diff
@@ -36,3 +36,13 @@
...
 *.out
 *.app
+
+/benchmark/*
+/CMakeFiles/*
+/tools/*
+
+*.txt
+*.cmake
+*.h
```
Comment (hellocsi) [inline .gitignore]: do not ignore these common files, *.h might be a cpp header file

File: src/layer/convolution.cpp [hunk 2]
```diff
@@ -147,10 +149,15 @@ int Convolution::create_pipeline(const Option& opt)
 
             float top_rescale = 1.f;
 
-            if (weight_data_int8_scales[n] == 0)
-                top_rescale = 0;
-            else
-                top_rescale = 1.f / (bottom_blob_int8_scale * weight_data_int8_scales[n]);
+            //if (weight_data_int8_scales[n] == 0)
```
Comment (hellocsi) [inline src/layer/convolution.cpp]: do not comment the useful code, only modify the code you must do

**Comment by nihui** [comment]
clos, as x86 avx2 optimization landed in the latest code

**Pull request closed without merging**